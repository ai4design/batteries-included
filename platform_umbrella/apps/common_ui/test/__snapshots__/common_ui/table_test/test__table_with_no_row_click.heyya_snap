<div class="px-4 overflow-auto sm:px-0">
  <table class="pc-table ">
  
    <thead>
      <tr class="pc-table__tr ">
  
        <th class="pc-table__th ">
  
          A
        
</th><th class="pc-table__th ">
  
          B
        
</th>
        
      
</tr>
    </thead>
    <tbody>
      <tr class="pc-table__tr group " id="-idx-0">
  
        <td class="pc-table__td ">
  
          100
        
</td><td class="pc-table__td ">
  
          test
        
</td>
        
      
</tr><tr class="pc-table__tr group " id="-idx-1">
  
        <td class="pc-table__td ">
  
          102
        
</td><td class="pc-table__td ">
  
          testing
        
</td>
        
      
</tr><tr class="pc-table__tr group " id="-idx-2">
  
        <td class="pc-table__td ">
  
          420
        
</td><td class="pc-table__td ">
  
          anyone there?
        
</td>
        
      
</tr>
    </tbody>
  
</table>
</div>