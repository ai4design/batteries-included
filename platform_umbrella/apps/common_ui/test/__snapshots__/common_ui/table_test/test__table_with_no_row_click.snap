"<div class=\"overflow-y-auto px-4 sm:overflow-visible sm:px-0\">\n  <table class=\"mt-4 w-full\">\n    <thead class=\"text-left text-base leading-6 text-gray-700\">\n      <tr>\n        <th class=\"p-0 pb-4 pr-6 text-pink-500\">\n          A\n        </th><th class=\"p-0 pb-4 pr-6 text-pink-500\">\n          B\n        </th>\n        \n      </tr>\n    </thead>\n    <tbody class=\"relative divide-y divide-gray-100 border-t border-gray-200 text-sm leading-6 text-gray-700\">\n      <tr id=\"-idx-0\" class=\"relative group hover:bg-gray-50\">\n        <td class=\"p-0 \">\n          <div>\n            <span class=\"absolute h-full w-4 top-0 -left-4 group-hover:bg-gray-50 sm:rounded-l-xl\"></span>\n            <span class=\"absolute h-full w-4 top-0 -right-4 group-hover:bg-gray-50 sm:rounded-r-xl\"></span>\n          </div>\n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative font-semibold text-gray-900\">\n              100\n            </span>\n          </div>\n        </td><td class=\"p-0 \">\n          \n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative \">\n              test\n            </span>\n          </div>\n        </td>\n        \n      </tr><tr id=\"-idx-1\" class=\"relative group hover:bg-gray-50\">\n        <td class=\"p-0 \">\n          <div>\n            <span class=\"absolute h-full w-4 top-0 -left-4 group-hover:bg-gray-50 sm:rounded-l-xl\"></span>\n            <span class=\"absolute h-full w-4 top-0 -right-4 group-hover:bg-gray-50 sm:rounded-r-xl\"></span>\n          </div>\n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative font-semibold text-gray-900\">\n              102\n            </span>\n          </div>\n        </td><td class=\"p-0 \">\n          \n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative \">\n              testing\n            </span>\n          </div>\n        </td>\n        \n      </tr><tr id=\"-idx-2\" class=\"relative group hover:bg-gray-50\">\n        <td class=\"p-0 \">\n          <div>\n            <span class=\"absolute h-full w-4 top-0 -left-4 group-hover:bg-gray-50 sm:rounded-l-xl\"></span>\n            <span class=\"absolute h-full w-4 top-0 -right-4 group-hover:bg-gray-50 sm:rounded-r-xl\"></span>\n          </div>\n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative font-semibold text-gray-900\">\n              420\n            </span>\n          </div>\n        </td><td class=\"p-0 \">\n          \n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative \">\n              anyone there?\n            </span>\n          </div>\n        </td>\n        \n      </tr>\n    </tbody>\n  </table>\n</div>"